app/views/protocols/form/study_form_sections/_study_information.html.haml

```diff
@@ -26,12 +26,13 @@
     .sub_text
       = t(:protocols)[:studies][:information][:subtext]
   .panel-body.container-fluid
-    .form-group.row
-      = form.label :research_master_id, class: 'col-lg-2 control-label rm-id' do
-        = link_to t(:protocols)[:studies][:information][:research_master_id], RESEARCH_MASTER_LINK
-      .col-lg-10
-        = form.text_field :research_master_id, class: 'form-control research-master-field'
-        = form.hidden_field :has_human_subject_info, value: false, class: 'has-human-subject-info'
+    - if RESEARCH_MASTER_ENABLED
```

Comment:
This needs to be done also in app/views/dashboard/protocols/form/study_form_sections/_study_information.html.haml

I think there's also code for RMID in app/views/protocols/form/_project_roles_fields.html.haml and the dashboard equivalent. Can you check that too?